zulipterminal/ui_tools/buttons.py

```diff
@@ -82,6 +82,10 @@ def update_widget(
         self.button_suffix.set_text(suffix)
         self._w.set_attr_map({None: text_color})
 
+    def set_highlight(self, style:str) -> None:
```

Comment:
This function isn't needed at all. Primarily because we don't want every child of TopButton to be highlightable. (Eg. EmojiButton)